.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.2.2
+ruby-2.3.0
```

Comment:
Probably best to use the lowest version that we want to support, which is `ruby-2.1.8` (Core). If it runs there, then it will run on later versions too.